apps/api/src/bootstrap.ts

```diff
@@ -103,6 +103,7 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
     .addTag('Integrations')
     .addTag('Layouts')
     .addTag('Workflows')
+    .addTag('Notification Templates')
```

Comment:
We still want to show the notification templates in our api reference, but all endpoints will have the deprecated tag.